File: app/src/main/java/com/slapshotapps/dragonshockey/models/SeasonStats.java
package com.slapshotapps.dragonshockey.models;

import androidx.annotation.Keep;
import java.util.ArrayList;
import java.util.List;

@Keep
public class SeasonStats {

    public List<GameStats> stats;

    public String seasonID;

    public SeasonStats() {
        stats = new ArrayList<>();
    }

    public SeasonStats(String seasonID) {
        this();
        this.seasonID = seasonID;
    }
}

File: app/src/main/java/com/slapshotapps/dragonshockey/models/SeasonStats.kt
package com.slapshotapps.dragonshockey.models

import androidx.annotation.Keep

@Keep
class SeasonStats() {

    @Keep
    @JvmField
    var stats = ArrayList<GameStats>()

    var seasonID = ""

    constructor(seasonID: String) : this() {
        this.seasonID = seasonID
    }
}
